some proposals / additions

Kr1ss-XD opened the pull request.

Kr1ss-XD committed:
use a POSIX shell instead of Bash

A simple Bourne shell (resp. Bash in POSIX mode) is perfectly alright
for our purpose, so no need to open a full Bash here.

Kr1ss-XD committed:
use the /dev/urandom stream rather than openssl

That way our script will run even on (tiny) devices where OpenSSL is
not installed.

Kr1ss-XD committed:
restrict input to positive integers

This commit adds a loop in order to repeat printing the prompt in case
something other than a positive integer value is entered, effectively
enforcing the correct input format.

Kr1ss-XD committed:
let the user enter how many passwords he needs

Here, we add another prompt to ask the user for the number of passwords
he actually wants.
This value is ensured to be a positive int the same way we do it for
PASS_LENGTH, and used as upper limit in the final genaration-/output-
loop.

AlexisAhmed commented:
> _No description provided._

Excellent suggestions, I will update the script as soon as possible 👍

Kr1ss-XD committed:
implement the input routine in form of a function

Since the prompting and check of the input values is done twice, let's
define a shell function for this tasks.
Maybe more features will be added to the script later, which could also
use the routine then.

Kr1ss-XD commented:
## Thx  :star2:
